tests/Unit/Framework/ActionTesting.hpp

```diff
@@ -673,7 +673,10 @@ class MockDistributedObject {
   using PhaseType =
       typename tmpl::front<phase_dependent_action_lists>::phase_type;
 
-  using all_cache_tags = Parallel::get_const_global_cache_tags<metavariables>;
+  using const_cache_tags = Parallel::get_const_global_cache_tags<metavariables>;
+  using mutable_cache_tags =
+      Parallel::get_mutable_global_cache_tags<metavariables>;
+  using all_cache_tags = tmpl::append<const_cache_tags, mutable_cache_tags>;
```

Comment:
I believe the same comment I made above about not placing the mutable_cache_tags in the DataBox apply here as well